app/src/testSystem/java/org/mozilla/tv/firefox/helpers/MockWebBackForwardList.kt

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.helpers
+
+import android.graphics.Bitmap
+import android.webkit.WebBackForwardList
+import android.webkit.WebHistoryItem
+
+/**
+ * An implementation of the [WebBackForwardList] abstract class for testing.
+ */
+class MockWebBackForwardList(
```

Comment:
SUPER optional nit: these terms are fairly fuzzy, and are frequently used interchangeably, but at least following the definition I'm aware of, this would be a 'fake' rather than a 'mock'.  https://martinfowler.com/articles/mocksArentStubs.html

Reply:
I'm part of the problem 😭